# part_pose: request of srv
geometry_msgs/Pose part_pose
---
# solutions returned: response of srv
int8 num_sols

# joint angles to reach part: request and reponse
ik_service/JointSolutions[8] joint_solutions